src/persist/src/nemesis/direct.rs

```diff
@@ -237,6 +237,7 @@ impl Direct {
         let contents = snap.read_to_end_flattened()?;
         Ok(ReadSnapshotRes {
             seqno: snap.seqno().0,
+            since: snap.since()[0],
```

Comment:
Let's not introduce any more of these, please :)

I think if we plumb the antichain, this just gets handled in `updates_eq` with something like:
```
for (_, t, _) in actual.iter_mut() {
    for ts in since.elements().iter() {
        if timely::order::PartialOrder::less_than(*t, *ts) {
            *t = *ts;
        }
    }
}
```

And then the optionality of updates_eq's since argument can be removed in favor of an empty antichain in place of None

Reply:
done!